test/requirements/python-requirements.txt

```diff
@@ -1,4 +1,6 @@
 backoff==2.2.1
 docker==7.1.0
+packaging==24.2
 pytest==8.3.4
 requests==2.32.3
+urllib3==2.3.0
```

Comment:
`urllib3` is explicitly installed to use `HAS_IPV6` instead of doing the dubious `from requests.packages.urllib3.util.connection import HAS_IPV6`.